zcash_primitives/src/sapling/zip32.rs

```diff
@@ -1193,506 +1151,431 @@ mod tests {
             },
             TestVector {
                 ask: Some([
-                    0x28, 0x2b, 0xc1, 0x97, 0xa5, 0x16, 0x28, 0x7c, 0x8e, 0xa8, 0xf6, 0x8c, 0x42,
```

Comment:
These updated test vectors come from https://github.com/zcash/zcash-test-vectors/pull/94.

Reply:
I was just about to ask about that. I also noticed that the URL at the top of this block of test vectors should now change.

Reply:
Yeah, it would have changed automatically had the test vector repo been updated when the repo was moved.